android/src/com/mapswithme/maps/Framework.java

```diff
@@ -61,6 +61,13 @@
     public boolean buildings;
   }
 
+  public static class RouteAltitudeLimits
+  {
+    public int minRouteAltitude;
```

Comment:
Why do we need this at all? I don't see places where this class is used. Moreover, it seems quite strange to me that the method named GenerateRouteAltitudeChart returns min and max altitudes of a route. min and max altitudes are properties of a current route, not a chart.